### PR TITLE
Initilize project

### PR DESCRIPTION
**Pull request by nguyenthanhtu95**

### CODE REVIEW

**Comment by nguyenthanhtu95** [comment]
ready

File: file1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+initilize project
```
Comment (khoapn-1940) [inline file1.txt]: Nhu shit

Comment (nguyenthanhtu95) [reply file1.txt]: fixed

**Comment by nguyenthanhtu95** [comment]
ready

**Pull request merged**